datajoint/autopopulate.py

```diff
@@ -265,13 +265,16 @@ def handler(signum, frame):
                 # spawn multiple processes
                 self.connection.close()  # disconnect parent process from MySQL server
                 del self.connection._conn.ctx  # SSLContext is not pickleable
-                with mp.Pool(
-                    processes, _initialize_populate, (self, jobs, populate_kwargs)
-                ) as pool, (
-                    tqdm(desc="Processes: ", total=nkeys)
-                    if display_progress
-                    else contextlib.nullcontext()
-                ) as progress_bar:
```

Comment:
Once changed pyproject.toml:require-python from >=3.8 to 3.9, black fails on format check, so update the format here, and only here